config-overrides.js

```diff
@@ -14,6 +14,12 @@ module.exports = {
 
     config.devServer = { host: 'local-ipv4' }
 
+    // If `TAURI_PLATFORM_TYPE` is present in the environment, we're building for Tauri.
```

Comment:
Perfect, this is the sort of thing I had in mind as far as separate entry points for tauri vs. web; it makes splitting out the code a lot cleaner. :+1: